frontend/src/views/variaveis/VariaveisSeries.vue

```diff
@@ -364,33 +379,59 @@ watch(SeriesAgrupadasPorAno, (novoValor) => {
               <th>
                 {{ dateToTitle(serie[$props.tipoDeValor as TiposValidos].periodo) }}
               </th>
+
               <td>
                 <Field
+                  v-if="seriesAgrupadas?.variavel?.variavel_categorica_id"
                   :disabled="modoDePreenchimento === 'valor_acumulado'"
-                  :aria-label="serie[$props.tipoDeValor as TiposValidos].periodo"
+                  :name="`${chave}[${idx}].${$props.tipoDeValor}.valor`"
                   class="inputtext light"
+                  as="select"
+                >
+                  <option value="">
+                    ---
+                  </option>
+
+                  <option
+                    v-for="item in opcoesVariaveisCategoricas"
+                    :key="item.id"
+                    :value="item.valor_variavel"
+                  >
+                    {{ item.titulo }} {{ item.descricao && `- ${item.descricao}` }}
```

Comment:
Curto circuitos são legais! 🤘

Reply:
Se a descrição for longa, vale a pena truncá-la. Já há ocorrências assim no sistema.